gradle/libs.versions.toml

```diff
@@ -0,0 +1,68 @@
+io-sentry-sentry-spring-boot-starter-jakarta = "io.sentry:sentry-spring-boot-starter-jakarta:6.19.0"
+net-javacrumbs-shedlock-shedlock-provider-jdbc-template = "net.javacrumbs.shedlock:shedlock-provider-jdbc-template:5.3.0"
+net-javacrumbs-shedlock-shedlock-spring = "net.javacrumbs.shedlock:shedlock-spring:5.2.0"
+nz-net-ultraq-thymeleaf-thymeleaf-layout-dialect = { module = "nz.net.ultraq.thymeleaf:thymeleaf-layout-dialect" }
+org-apache-santuario-xmlsec = "org.apache.santuario:xmlsec:3.0.2"
+org-awaitility = "org.awaitility:awaitility:4.2.0"
+org-bouncycastle-bcprov-jdk15on = "org.bouncycastle:bcprov-jdk15on:1.70"
+org-flywaydb-flyway-core = "org.flywaydb:flyway-core:9.18.0"
+org-jetbrains-kotlin-kotlin-bom = { module = "org.jetbrains.kotlin:kotlin-bom", version.ref = "kotlin" }
+org-jetbrains-kotlin-kotlin-reflect = { module = "org.jetbrains.kotlin:kotlin-reflect" }
+org-jetbrains-kotlin-kotlin-stdlib-jdk8 = { module = "org.jetbrains.kotlin:kotlin-stdlib-jdk8" }
+org-jsoup = "org.jsoup:jsoup:1.16.1"
+org-postgresql = "org.postgresql:postgresql:42.6.0"
+org-springdoc-springdoc-openapi-starter-webmvc-api = { module = "org.springdoc:springdoc-openapi-starter-webmvc-api", version.ref = "org-springdoc" }
+org-springdoc-springdoc-openapi-starter-webmvc-ui = { module = "org.springdoc:springdoc-openapi-starter-webmvc-ui", version.ref = "org-springdoc" }
+org-springframework-boot-spring-boot-configuration-processor = { module = "org.springframework.boot:spring-boot-configuration-processor" }
+org-springframework-boot-spring-boot-devtools = { module = "org.springframework.boot:spring-boot-devtools" }
+org-springframework-boot-spring-boot-starter-actuator = { module = "org.springframework.boot:spring-boot-starter-actuator" }
+org-springframework-boot-spring-boot-starter-aop = { module = "org.springframework.boot:spring-boot-starter-aop" }
+org-springframework-boot-spring-boot-starter-data-jpa = { module = "org.springframework.boot:spring-boot-starter-data-jpa" }
+org-springframework-boot-spring-boot-starter-security = { module = "org.springframework.boot:spring-boot-starter-security" }
+org-springframework-boot-spring-boot-starter-test = { module = "org.springframework.boot:spring-boot-starter-test" }
+org-springframework-boot-spring-boot-starter-thymeleaf = { module = "org.springframework.boot:spring-boot-starter-thymeleaf" }
+org-springframework-boot-spring-boot-starter-validation = { module = "org.springframework.boot:spring-boot-starter-validation" }
+org-springframework-boot-spring-boot-starter-web = { module = "org.springframework.boot:spring-boot-starter-web" }
+org-springframework-boot-spring-boot-starter-webflux = { module = "org.springframework.boot:spring-boot-starter-webflux" }
+# @pin
+org-springframework-cloud-spring-cloud-starter-kubernetes-client-config = "org.springframework.cloud:spring-cloud-starter-kubernetes-client-config:2.1.3"
+org-springframework-data-spring-data-redis = "org.springframework.data:spring-data-redis:3.0.5"
```

Comment:
Why does this Spring dependency have a pinned version and does not use the version provided by Spring Boot?

Reply:
This is the update that fails after deployment, see https://github.com/digitalservicebund/useid-backend-service/pull/232

Reply:
I'm talking about the `data-redis` package

Reply:
> How and where would we use the dependencyUpdates task? Do we include it into our pipeline? Lefthook? Or is this something we do manually on demand?

I run `dependencyUpdates` only locally, manually, to get a report on what updates are available, and `versionCatalogUpdate` to apply the updates. I currently don't see a need to include the task in any pipeline, and dependabot should still provide PRs as usual (only updating `libs.versions.toml` instead of `build.gradle.kts`).

Reply:
But you're right, we can omit the version and leave it to the spring dependency plugin to determine the version, it will pick 3.0.5, the working version.

Reply:
Alright 👍